packages/cdk-assets/lib/private/docker.ts

```diff
@@ -54,6 +55,7 @@ export class Docker {
     const buildCommand = [
       'build',
       ...flatten(Object.entries(options.buildArgs || {}).map(([k, v]) => ['--build-arg', `${k}=${v}`])),
+      ...flatten(Object.entries(options.buildSecrets || {}).map(([k, v]) => ['--secret', `id=${k},${v}`])),
```

Comment:
@danwiltshire sorry for reaching you out in a already merged PR, but shouldn't it be `id=${k},src=${v}` instead? It seems to be that docker cli always expect it to be a key pair with key and value.

Reply:
Thanks for catching this @abilun! Would you be able to submit a follow-up PR to fix this?

Reply:
Hello, when used with `DockerBuildSecret`, its method(s) specify the `src=`.

`DockerBuildSecret.fromSrc(...)`

Happy for any changes to be made though.